docs/all.md

```diff
@@ -145,26 +145,20 @@ $ kubeaudit all -k "config.yaml" -f "auditors/all/fixtures/audit_all_v1.yml"
 
 ### Example with Flags
 
-The behaviour of the `all` command can also be customized by using flags. The `all` command supports all flags supported by invididual auditors (see the individual [auditor docs](/README.md#auditors) for all the flags). For example, the `caps` auditor supports specifying capabilities to drop with the `--drop/-d` flag so this flag can be used with the `all` command:
+The behaviour of the `all` command can also be customized by using flags. The `all` command supports all flags supported by individual auditors (see the individual [auditor docs](/README.md#auditors) for all the flags).
+
+For example, the `--allowAddList` flag will make the `caps` auditor turn off reports for the listed capabilities (comma separated values).
 
 ```
-kubeaudit all -f "auditors/all/fixtures/audit_all_v1.yml" --drop "AUDIT_WRITE"
+kubeaudit all -f "manifest.yml" --allowAddList "AUDIT_WRITE"
 ```
 
-### Example with Kubeaudit Config and Flags
+Since `AUDIT_WRITE` is in the Capabilities Allow Add List, kubeaudit won't report errors, as an effect of passing that value to the `--allowAddList` flag.
 
-Passing flags in addition to the config will override the corresponding fields from the config. For example, if the capabilities to drop are specified with the `--drop/-d` flag:
+Another example, would be using the `--memory` flag (supported by the `limits` auditor):
 
 ```
-kubeaudit all -f "auditors/all/fixtures/audit_all_v1.yml" --drop "AUDIT_WRITE"
-```
-
-And they are also specified in the Kubeaudit config file:
-
-```yaml
-auditors:
-    capabilities:
-        drop: ["CHOWN", "MKNOD]
+kubeaudit all -f "manifest.yml" --memory 200
 ```
 
-The capabilities specified by the flag will take precedence over those specified in the config file resulting in only `AUDIT_WRITE` being dropped.
+Here, if the memory specified is higher than 200, `kubeaudit` will report that the memory limit was exceeded.
```

Comment:
@genevieveluyt what do you think of rephrasing here and also adding the `limits` as an extra example, as you suggested?

Reply:
I think just one would be sufficient. The more duplicated documentation we have the more likely it'll get out of sync. Wdyt?